### DISCUSSION TITLE
Possible bug: "group:nodeJs" preset's `matchPackagePatterns` option value

### DISCUSSION THREAD

**Discussion by chalkygames123** [renovatebot/renovate#11819] [Request Help] [answered]
Just in case it is a bug, the regular expression in the `matchPackagePatterns` option of the ["group:nodeJs"](https://docs.renovatebot.com/presets-group/#groupnodejs) preset currently starts with a slash.

https://github.com/renovatebot/renovate/blob/4303063b2ab77e5680d96fa659dd146195f781d7/lib/config/presets/internal/group.ts#L41

Is this correct? It looks like an invalid value, given that other occurrences in the docs usually start with a literal character or caret `^`.

**Comment by rarkins** [comment, marked as answer]
It's intended to match Docker images like foo/node, my/node, etc

**Reply by chalkygames123** [reply]
Got it, thank you for the confirmation :)
